### ISSUE TITLE
PhET Menu - Keyboard focus highlight thin on bottom

### ISSUE THREAD

**Issue by terracoda** [phetsims/john-travoltage#217]
Visually, keyboard focus highlight is slightly cut off (slightly thinner) on bottom border.

MacOs 10.12.3
Safari 10.0.3

**Comment by jessegreenberg** [comment]
Thanks @terracoda! I am only seeing this for the "PhET Menu" button, not the "Keyboard Help Button", is that what you see too?

It looks like there is a bit of space below the branding image.

**Comment by jessegreenberg** [comment]
Ready for review in the next version, should be deployed shortly.

**Comment by terracoda** [comment]
@jessegreenberg, Yes, I only see it cut off on the PhET Menu button.

**Comment by jessegreenberg** [comment]
Ok, thanks!

**Comment by jessegreenberg** [comment]
@terracoda @emily-phet do you like how the focus highlight looks on the two navigation bar buttons in this version?

http://www.colorado.edu/physics/phet/dev/html/john-travoltage/1.3.0-dev.28/john-travoltage_en.html

**Comment by emily-phet** [comment]
@jessegreenberg I think this looks great!

**Comment by jessegreenberg** [comment]
Ok, thanks!  Closing.
